<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ZCHManage</class>
 <widget class="QDialog" name="Dialog_ZCHManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>421</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>站存车信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 210, 218);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,6,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>修改时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>发送时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>接收时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tableView_2"/>
         </item>
         <item>
          <widget class="QTableView" name="tableView_3"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>存车信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>编辑站存车</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>24</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>类别</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>60</width>
              <height>20</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>20</y>
              <width>24</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>车种</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_2">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>20</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>20</y>
              <width>48</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>作业说明</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_3">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>20</y>
              <width>60</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>24</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>辆数</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>60</y>
              <width>48</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>运用标记</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>54</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>说明/去向</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_5">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>60</y>
              <width>151</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>61</y>
              <width>60</width>
              <height>18</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>100</y>
              <width>241</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="unlockButton_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>140</y>
              <width>70</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
            </property>
            <property name="text">
             <string>插入</string>
            </property>
           </widget>
           <widget class="QPushButton" name="unlockButton_4">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>140</y>
              <width>70</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
            </property>
            <property name="text">
             <string>增加</string>
            </property>
           </widget>
           <widget class="QPushButton" name="unlockButton_6">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>140</y>
              <width>70</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
           <widget class="QPushButton" name="unlockButton_5">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>140</y>
              <width>70</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
            </property>
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>合计:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>728</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="lockButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
        </property>
        <property name="text">
         <string>上报调度所</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unlockButton_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
